sdcm/sct_events/setup.py

```diff
@@ -73,6 +73,13 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
     EventsSeverityChangerFilter(new_severity=Severity.WARNING,
                                 event_class=DatabaseLogEvent.RUNTIME_ERROR,
                                 regex='.*view update generator not plugged to push updates').publish()
+
+    # cause of issue https://github.com/scylladb/scylla-cluster-tests/issues/6119
+    EventsSeverityChangerFilter(new_severity=Severity.WARNING,
+                                event_class=DatabaseLogEvent.RUNTIME_ERROR,
+                                regex=r'.*sidecar/controller.go.*std::runtime_error '
+                                      r'\(Operation decommission is in progress, try again\)').publish()
```

Comment:
Shouldn't the regex end with `.*`? Or add one more symbol `]`?

Reply:
unittest show this line was caught, good enough for me

Reply:
I learnt the hard way we should check any of those.

cause we tend todo them multiple times